python/main.py

```diff
@@ -25,23 +25,29 @@ def root():
     if request.method == 'GET':
         # get url params: request.args.get(KEY)
         # This request is currently not being used
-
-        query = 'hello'
         json_in = '{"captions": [{"text": "hello, world"}, {"text": "forget me"}]}'
-        res = query_phrase(query, json_in)
-        print("RES: " + str(res))
+        sentance = "this is a really cool sentance to analyze"
+
+        # query = 'unrelated'
+        while True:
```

Comment:
Could u explain a little why are we using ```while True```?

Reply:
This `GET` request is just for testing purposes, I run this locally and ask for queries input in an inf loop to test.

Reply:
This has been changed, GET request are now in use